hack/make-rules/test-cmd.sh

```diff
@@ -101,7 +121,7 @@ function run_kube_controller_manager() {
 # Exports:
 #   SUPPORTED_RESOURCES(Array of all resources supported by the apiserver).
 function create_node() {
-  kubectl create -f - -s "http://127.0.0.1:${API_PORT}" << __EOF__
+  kubectl create -s "https://127.0.0.1:${SECURE_API_PORT}" --insecure-skip-tls-verify --token=admin-token -f - << __EOF__
```

Comment:
I expected to set $KUBECONFIG pointing at a file containing credentials so we don't have to set the token in the invocation